src/components/GameInfoRow.js

```diff
@@ -67,6 +87,16 @@ function GameInfoRow({ gameId, onClick }) {
           <TableCell>
             {game.users ? Object.keys(game.users).length : 0}
           </TableCell>
+          <TableCell
+            style={{
+              color:
+                colors[modeMapping[game.mode].color][
```

Comment:
These usages of `game.mode` can be null for older games, and I would prefer not to make mass changes to the database if not necessary. Maybe set a local variable to `modeMapping[game.mode || "normal"]` ?

![image](https://user-images.githubusercontent.com/7550632/103449895-3c81f800-4c74-11eb-9ce2-1bc2233958a8.png)